yarn.lock

```diff
@@ -6406,11 +6406,6 @@ generate-changelog@1.8.0:
     commander "^2.9.0"
     github-url-from-git "^1.4.0"
 
-gensequence@^3.1.1:
-  version "3.1.1"
-  resolved "https://registry.yarnpkg.com/gensequence/-/gensequence-3.1.1.tgz#95c1afc7c0680f92942c17f2d6f83f3d26ea97af"
-  integrity sha512-ys3h0hiteRwmY6BsvSttPmkhC0vEQHPJduANBRtH/dlDPZ0UBIb/dXy80IcckXyuQ6LKg+PloRqvGER9IS7F7g==
-
```

Comment:
humm this is weird, I don't see any changes in the package.json 🤔

Reply:
i'm not sure, I just did yarn install 🤔

Reply:
🤷  maybe try to re-run `yarn` locally. 
If there are no other changes let's keep it this way then I guess